### ISSUE TITLE
The mod user-interface should reflect what's on curseforge

### ISSUE THREAD

**Issue by Xiaminou** [Watchful1/CurseAppFeatureTracker#7]
When you select a mod in the list (either from a modpack or from the browser) you're put on a page that's supposed to have all the informations about said mod, except it doesn't.

The overview is fine, but it could have the comments posted on curseforge, wouldn't hurt.
The changelog is bad, you would assume that it's the changelog for the latest version of the mod, but it's often not the case. This should be deleted.

Basically delete everything and make it like it is on Curseforge.

**Comment by Watchful1** [comment]
Not sure I follow. Comments would be useful, but it looks to me like the changelog is indeed the most recent for a mod. Do you have an example otherwise?

Though it does look like you can't access changelogs for older versions, which would be useful.

**Comment by Xiaminou** [comment]
[Advanced Generator](http://minecraft.curseforge.com/projects/advanced-generators/files), the changelog is for the 1.7.10 release when my modpack is 1.8.9. So it's completely irrelevant.

And what I mean is add all the different tabs that you have on Curseforge into the Curse Voice minecraft plugin. Images, Issues, Wiki, Source, Relations. You could also add a tab for mods by the same author.

**Comment by Watchful1** [comment]
Hmm, good point. Looks like it shows the changelogs for the recommended version. I still like your idea of getting rid of it entirely and showing changelogs per version.
